# HTGTS_Bcell_Analysis
Source code for genomic event analysis tools (GEAT) developed in our laboratory to perform the analysis is available at https://github.com/geatools/geat
The tools can be initialized through platform-specific scripts: script.cmd for Windows and script.sh for Linux/OS X. Executing these scripts launches a Docker container that is accessible via localhost:8888. This process automatically initiates a Jupyter Lab environment within the Docker container. The environment shares a folder named sharedFolder with the host system, ensuring seamless data integration and management.
The data used in these analyses are available on GEO (Gene Expression Omnibus) and should be placed in the same folder as the script (./Data) for optimal processing and analysis. In the supplementary table (dataStructure.txt ) there is a file with the final structure of the sharedFolder. Use the command “ less -R dataStructure.txt “ to explore it with the correct colors and format. 
